src/net.h

```diff
@@ -363,6 +365,8 @@ class CNode
     CBloomFilter* pfilter;
     int nRefCount;
     NodeId id;
+
+    std::vector<unsigned char> vchKeyedNetGroup;
```

Comment:
uint64_t please.   The probability that two distinct net-groups share a 64-bit hash is negligible, and if it ever happens its harmless (and would have no effect at all unless it's a collision with one of the four lowest ones that are in use).